layers/meta-tegrademo/recipes-bsp/tegrademo-devicetree/tegrademo-devicetree_1.0.bb

```diff
@@ -0,0 +1,29 @@
+DESCRIPTION = "Meta-tegrademo example device trees for out-of-tree builds."
+HOMEPAGE = "https://github.com/OE4T"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+inherit devicetree
+
+COMPATIBLE_MACHINE = "(tegra)"
+
+SRC_URI = "\
+    file://tegra194-p2888-0001-p2822-0000-oe4t.dts \
+    file://tegra194-p3668-all-p3509-0000-oe4t.dts \
+"
+
+DEPENDS += "linux-tegra"
+
+KERNEL_INCLUDE = " \
+    ${STAGING_KERNEL_DIR}/nvidia/soc/tegra/kernel-include \
+    ${STAGING_KERNEL_DIR}/nvidia/platform/tegra/common/kernel-dts \
+    ${STAGING_KERNEL_DIR}/nvidia/soc/t19x/kernel-include \
+    ${STAGING_KERNEL_DIR}/nvidia/soc/t19x/kernel-dts \
+    ${STAGING_KERNEL_DIR}/nvidia/platform/t19x/common/kernel-dts \
+    ${STAGING_KERNEL_DIR}/nvidia/platform/t19x/galen/kernel-dts \
+    ${STAGING_KERNEL_DIR}/nvidia/platform/t19x/jakku/kernel-dts \
+    ${STAGING_KERNEL_DIR}/nvidia/platform/t19x/mccoy/kernel-dts \
+"
+
+# Straight from arch/arm64/boot/dts in kernel source tree
+DTC_PPFLAGS:append = " -DLINUX_VERSION=504 -DTEGRA_HOST1X_DT_VERSION=1"
```

Comment:
LINUX_VERSION should not be `510` ?

Reply:
No, this is correct... see [this](https://github.com/OE4T/linux-tegra-5.10/blob/63c149056a7ef7bf146a747e7c8a179c1aaf72f7/arch/arm64/boot/dts/Makefile#L5)

Reply:
Sorry for the duplicate comment, Github gave me an error the first time around, so I thought it didn't take.